zgw_auth_backend/authentication.py

```diff
@@ -41,15 +41,20 @@ def authenticate(self, request: Request):
             msg = _("Invalid 'user_id' claim. The 'user_id' should not be empty.")
             raise exceptions.AuthenticationFailed(msg)
 
-        return self.authenticate_user_id(user_id)
+        email = auth.payload.get("zds", {}).get("user_email", "")
```

Comment:
I would advise to follow the structure of OpenID standard claims here: https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims

So, `email` should be a claim directly in the payload, not scoped under `zds`. The `zds` payload key is deprecated in gemma-zds-client as well.

I'll continue working on gemma-zds-client a bit more for a 1.0 release, which drops all the deprecated things. That will make it easier to add extra claims to `ClientAuth` as well.